### PR TITLE
Restaurant homepage

### PR DESCRIPTION
**Pull request by R4Ajeti**
Dynamically rendering a restaurant homepage! We are using only JavaScript to generate the entire contents of the website! 
We are using webpack because: 
1. We need to import stuff from place,
2. We know we need to concatenate and compress our JavaScript anyway, and managing load order isn't something you wanna handle manually,
3. Of npm,
4. We're sure it performs fancy magic that results in good performance-related things for our websites,
5. There is a hive mind in this industry and it leads to a lot of us hopping on the trains with the most people on them, and people are hanging out of the windows of the webpack train.

### CODE REVIEW

File: .stickler.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# add the linters you want stickler to use for this project
+linters:
+    eslint:
+      # indicate where is the config file for stylelint
+      config: './.eslintrc.json'
+  
+  # add the files here you want to be ignored by stylelint
+  files:
```
Comment (jofer86) [inline .stickler.yml]: This file is not parsing properly due to an indentation issue you have an indentation on this line which is making the yml file not to parse, just alingn this line with line one, that is no tabs or spaces in the begginig of this line and the issue shall be taken care off and we will be alble to see what stickler has to say.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello @R4Ajeti , I'm here to review this PR, let's take a look shall we :eye: :eye: 

- All is fine but your stickler check is not passing see: 
image
- Check the comment below for a further explanation. Fix and request another code review.
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

File: .stickler.yml (hunk 1 above)
Comment (R4Ajeti) [reply .stickler.yml]: Hi @jofer86  thanks for reviewing my code,

You mean remove indention on line 5 right?

**Review by bellom — APPROVED** [review]
## Hi @R4Ajeti  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**